Jenkinsfile

```diff
@@ -67,9 +67,12 @@ pipeline {
                 stash name: 'repo-amd64',includes: 'rook-amd64.tar,build/common.sh,_output/tests/linux_amd64/,_output/charts/,tests/scripts/'
                 script{
                     def data = [
-                        "gce_1.6": "v1.6.7",
-                        "aws_1.7": "v1.7.8",
-                        "gce_1.8": "v1.8.2",
+                        "gce_1.6.5": "v1.6.5",
+                        "gce_1.6": "v1.6.13",
```

Comment:
could we put the full version in the display for all of them? some of them are missing the hotfix version

Reply:
I have updated the labels to gce_1.6_latest,aws_1.7_latest and aws_1.8_latest. That way next time there is a new k8s version, we need to just updated the jenkins file and not the config on jenkins site.  When we integrate 1.9 for the first build, i will add aws_1.9.0 lable to jennkis config and the aws or gce 1.9_latest for all other 1.9 versions.